### ISSUE TITLE
Where is the report with "Offer Start Date" for a listing?

### ISSUE THREAD

**Issue by johnkw** [amzn/selling-partner-api-docs#440]
Where is the report with "Offer Start Date" for a listing?  I would expect this to be part of `GET_MERCHANT_LISTINGS_ALL_DATA`, but it's not there.

There are some other key fields like "Max Order Quantity", and sales/MSRP fields, that I would have expected in that same report, but I can't find any of them.

(See also bug #421.)

**Comment by johnkw** [comment]
I just found these key fields are all in the Category Listings Report. "Offer Start Date" is called `product_site_launch_date`.

Unfortunately the Category Listings Report" is weirdly painful to get, so that's not a good option, unless that report can be opened up to be easier to access.

**Comment by BlinglingGirl** [comment]
hey , I have a problem about the type of report `GET_MERCHANT_LISTINGS_ALL_DATA` .

I want to get my seller's listing.

 should I use the report-api to get ? Do you have any better suggestions？

**Comment by johnkw** [comment]
That doesn't have anything to do with this bug report - please delete that comment and open a new bug report if needed.

**Comment by johnkw** [comment]
As per https://github.com/amzn/selling-partner-api-docs/blob/main/references/listings-items-api/listingsItems_2021-08-01.md this appears to be working.  The attribute locations and values still need to be documented, but it is currently showing up as:

`'attributes' > 'purchasable_offer' > 'start_at'`

**Comment by johnkw** [comment]
The API supports this now, but the github bug is not resolved because there's no documentation for this.

**Comment by johnkw** [comment]
Turn this stupid bot off and resolve the github tickets as the actual issues are fixed. @adrian-amz @amazon-auto @chapmanjw @amvreddy-amazon @jenilew

**Comment by johnkw** [comment]
Please turn this horrible bot off and only close the github tickets as the actual issues are fixed. @adrian-amz @amazon-auto @chapmanjw @amvreddy-amazon @jenilew @parvathm

**Comment by cohlar** [comment]
> As per https://github.com/amzn/selling-partner-api-docs/blob/main/references/listings-items-api/listingsItems_2021-08-01.md this appears to be working. The attribute locations and values still need to be documented, but it is currently showing up as:
> 
> `'attributes' > 'purchasable_offer' > 'start_at'`

@johnkw are you sure?
Based on my current tests, `'attributes' > 'purchasable_offer' > 'start_at'` doesn't update the "Offer Start Date" field

**Comment by johnkw** [comment]
Yes, the documentation is now at https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#getlistingsitem and still needs to be fixed to actually, well, document, these calls.  But that is the same field, and FYI to all, it shows up in Seller Central after about 30 minutes.  No explanation on the ridiculous delay.

**Comment by cohlar** [comment]
Thanks for your response. Actually I've played with it a little more and it worked for me in the US and France (the field there is named "Offering Release Date" in Seller Central), but not in Italy and Spain (the field there is named "Offer Start Date" in Seller Central)
